modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -1,270 +1,306 @@
+The *API Details Button* component provides portal users with details about an API.
+
+You can specify values for the following parameters of the *API Details Button*:
 
-API:: Choose the API to display.
-Button Label:: Set the button label. Default is `API Details`.
-Button Align:: Set button alignment to left, right, or center of the UI field.
-API Name:: Add the API name to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
-API Version:: Add the API version to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
+API:: Specifies the API to display.
+Button Label:: Specifies the button label. Default is `API Details`.
+Button Align:: Specifies button alignment as left, right, or center of the UI field.
+API Name:: Adds the API name to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
+API Version:: Adds the API version to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
 
 == API Documentation Viewer
 
-The API documentation viewer component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+The *API Documentation Viewer* component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+
+You can specify values for the following parameters of the *API Documentation Viewer*:
 
-API:: Choose the API.
-Section (Optional):: Set the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
-Selected Page (Optional):: Set the page to open.
+API:: Specifies the API.
+Section (Optional):: Selects the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
+Selected Page (Optional):: Selects the page to open.
 
 == API Header
 
-The API header component displays the API Header including icon, name, and description.
+The *API Header* component displays the API header, including icon, name, and description.
 
-API:: Choose the API to show.
+You can specify values for the following parameters of the *API Header*:
+
+API:: Specifies the API to show.
 
 == Application Analytics Graph
 
-The application analytics graph component displays statistics about the use of an application.
+The *Application Analytics Graph* component displays statistics about the use of an application, such as traffic, response time, and application health.
+
+You can specify values for the following parameters of the *Application Analytics Graph*:
 
-Application:: Choose the application to show.
+Application:: Specifies the application to show.
 
 == Application Details
 
-The application details component displays details about an application.
+The *Application Details* component displays details about an application.
 
 // check component description
```

Comment:
Internal comments should not appear in any text that is kept in the public repo. GLOBALLY replace these with Jiras going forward.

Reply:
Removed to Jiras.